test/dynamo/test_package.py

```diff
@@ -30,6 +30,50 @@
 def compute_loss_helper(x):
     return reduce_to_scalar_loss(x)
 
+class _tempTensorSamplerForQualName:
+    def __init__(self, val, mask, prob):
+        self.val = val
+        self.mask = mask
+        self.prob = prob
+
```

Comment:
Can we add some instance method that is used in these tests as well? That can be a seperate test case, but we should check how these interplay

Reply:
I have updated the unit-test :)